src/ErrorHandler.php

```diff
@@ -83,8 +83,6 @@ final class ErrorHandler
      *
      * @param callable $callback           The callback that will be invoked in case an error is caught
      * @param int      $reservedMemorySize The amount of memory to reserve for the fatal error handler
-     *
-     * @throws \ReflectionException
      */
     private function __construct(callable $callback, int $reservedMemorySize = self::DEFAULT_RESERVED_MEMORY_SIZE)
```

Comment:
Since the constructor is `private` remove the default value for the `$reservedMemorySize` argument